rpc/query_server.go

```diff
@@ -578,6 +578,27 @@ func (s *QueryServer) ContractEvents(
 	}, nil
 }
 
+func (s *QueryServer) GetContractRecord(contractAddrStr string) (*types.ContractRecordResponse, error) {
+	contractAddr, err := loom.ParseAddress(contractAddrStr)
+	if err != nil {
+		return nil, err
+	}
+	snapshot := s.StateProvider.ReadOnlyState()
+	defer snapshot.Release()
+
+	reg := s.CreateRegistry(snapshot)
+	rec, err := reg.GetRecord(contractAddr)
+	if err != nil {
+		return nil, err
+	}
+	k := &types.ContractRecordResponse{
+		ContractName:    rec.GetName(),
+		ContractAddress: rec.GetAddress(),
+		CreatorAddress:  rec.GetOwner(),
```

Comment:
You can just use `rec.Name` etc. here, no need to use the getters. The getters only deal with the case where `rec == nil`, but that won't happen here.